### ISSUE TITLE
New label

### ISSUE THREAD

**Issue by Mrdeer3935** [uni-medical/STU-Net#22]
Thank you for sharing! Can I apply the pre-trained parameters and architecture of STU to a new dataset (not the previously mentioned 104 types of segmentation), as the features needed for segmentation are generic and I only need to fine-tune the output layer?
